index.html

```diff
@@ -188,7 +188,7 @@ <h1>The Registry</h1>
 
     <tr>
       <td>
-          EdDsaSAPublicKeySecp256k1
+          ECDSASecp256k1PublicKey2018
```

Comment:
Capitalization nitpick: This may be better as EcDsaSecp256k1PublicKey2018 ... I'll merge anyway, but we may want to discuss this before anything is finalized. That's typically how capitalization is done in JSON-LD... but I don't know if the 's', 'p', or the 'k' should be cap'd or lowercased.

Reply:
-1 to this change, not because of the correction (which I agree is needed), but because it should be what was recommended on the mailing list because it's more consistent with other key types:

`Secp256k1VerificationKey2018`